src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,41 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends AbstractManager {
+    private final String position ="manager";
+
+    @Override
+    public String getPosition() {
```

Comment:
Regular methods should be defined below constructor. Especially getters and setters which usually are defined at the end of a file.